src/compiler/usecheck.rs

```diff
@@ -65,6 +65,17 @@ fn remove_present_atoms(envlist: &mut HashMap<Vec<u8>, Vec<u8>>, args: Rc<SExp>)
     }
 }
 
+/// Given a CompilerOpts and a compiled program CompileForm, produce the set of
+/// eligible parameters to the program which, after expanding the complete program
```

Comment:
```suggestion
/// formal parameters to the program which, after expanding the complete program
```

`eligible` seems a bit redundant, and `formal` shows we want the symbol, not the value.